os/booting-on-azure.md

```diff
@@ -49,10 +49,10 @@ startup, and more via cloud-config. Jump over to the [docs to learn about the
 supported features][cloud-config-docs]. Cloud-config is intended to bring up a
 cluster of machines into a minimal useful state and ideally shouldn't be used
 to configure anything that isn't standard across many hosts. Once an instance
-is provisioned on Azure, the cloud-config cannot be modified.
+is provisioned on Microsoft Azure, the cloud-config cannot be modified.
 
-You can provide raw cloud-config data to CoreOS
-<a href="#via-the-cross-platform-cli">via the Azure cross-platform CLI</a>.
+You can provide raw cloud-config data to CoreOS <a href="#via-the-cross-
```

Comment:
TODO: joshix: [Style guide](https://github.com/coreos/docs/blob/master/STYLE.md) fixes: Unwrap lines. Links should be markdown not raw html.